include/boost/thread/detail/thread.hpp

```diff
@@ -588,7 +591,6 @@ namespace boost
 #endif
 
 #if defined BOOST_THREAD_PROVIDES_INTERRUPTIONS
-        void BOOST_THREAD_DECL interruption_point();
```

Comment:
Wondering if the file shouldn't contains the declarations related to interruptions :)

Reply:
In this case the file name should be interrupt.hpp